helpers/test-data-generator.js

```diff
@@ -15,17 +15,25 @@ import {
   RATING_2_SALT,
   MANAGER_ROLE,
   EVALUATOR_ROLE,
-  WORKER_ROLE,
-  SPECIFICATION_HASH
+  WORKER_ROLE
 } from "../helpers/constants";
 import { currentBlockTime, createSignatures } from "../helpers/test-helper";
 
 const IColony = artifacts.require("IColony");
 const Token = artifacts.require("Token");
 
 export async function setupAssignedTask({ colonyNetwork, colony, dueDate, domain = 1, skill = 0, evaluator = EVALUATOR, worker = WORKER }) {
-  await colony.makeTask(SPECIFICATION_HASH, domain);
+  // By using a random specification hash rather than SPECIFICATION_HASH, we can create tasks in parallel and pick up
+  // the correct taskID afterwards.
```

Comment:
I don't get what or why we are doing this

Reply:
I want to make the client tests quicker to run, which currently have a lot of setup with multiple tasks being created to give users tokens and staking (anywhere between 3 and 11 times, I believe). If we can do that setup in parallel, I'm hoping it'll be quicker. This prepares for that; #203 is the remaining piece of the puzzle.